components/carousel/module/CarouselInfo.vue

```diff
@@ -26,12 +32,19 @@ import type { CarouselNFT } from '@/components/base/types'
 
 import { useCarouselUrl } from '../utils/useCarousel'
 
-defineProps<{
+const props = defineProps<{
   item: CarouselNFT
 }>()
 
-const { urlPrefix } = usePrefix()
 const { urlOf } = useCarouselUrl()
 const url = inject('itemUrl') as string
 const isCollection = inject('isCollection')
+const chainName = computed(() => {
+  const name = {
+    rmrk: 'RMRK',
+    snek: 'Basilisk',
```

Comment:
Why renaming `snek`nft with `Basilisk` 👀

Reply:
tbh, I still don't know the difference 🙈
what was the correct name?

Reply:
@petersopko  Could you pls answer this question?

Reply:
I think we are going to be eventually sunsetting /snek route as this is only the final testnet where we could mirror the functionality of the actual Basilisk which is coming in live at 
- #4030

Reply:
From https://github.com/kodadot/nft-gallery/pull/4011#pullrequestreview-1119281239
> note: would it be possible to change the text of the chain to be capitalized or uppercase? (snek should be Basilisk?)

Since it's going to be on Basilisk parachain wouldn't it be wiser to rename it?

Reply:
> Since it's going to be on Basilisk parachain wouldn't it be wiser to rename it?

it is Basilisk now, that's what @Jarsen136 was asking about
![image](https://user-images.githubusercontent.com/73316633/192493456-d5008fb5-7a9e-4c4a-b6e9-e3c59eaf59f9.png)